docker/Dockerfile

```diff
@@ -11,5 +11,9 @@ RUN mkdir -p $GOPATH/src/github.com/algorand
 WORKDIR $GOPATH/src/github.com/algorand
 RUN git clone https://github.com/algorand/go-algorand
 WORKDIR $GOPATH/src/github.com/algorand/go-algorand
+RUN apt-get -y install sudo
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get -y install tzdata
+RUN apt-get -y install bsdmainutils
 RUN git checkout master && scripts/configure_dev.sh && make
```

Comment:
This should be changed to `make install` so it's usable immediately.

Reply:
I believe the make install step will fail, when the dependency checker checks for the depedencies from configure-dev_deps.sh. Go get -u <path> does not actually install the dependencies it fetches to $GOPATH/bin so they are not found.